syntax = "proto3";

package response;


message ReadReponse {
    string data = 1;
}